src/main/webapp/features/screens/game/view.html

```diff
@@ -1,48 +1,3 @@
-<div class="row">
+<ct-game-room ng-if="game.isNew()" game="game"></ct-game-room>
```

Comment:
why? It's not a game details, it's "game room". We can't see at the same time "game room" and "game playing"